app-android/src/main/java/io/github/droidkaigi/confsched2023/KaigiApp.kt

```diff
@@ -169,9 +171,15 @@ private fun NavGraphBuilder.mainScreen(
             )
             // For KMP, we are not using navigation abstraction for contributors screen
             composable(contributorsScreenRoute) {
+                val lifecycleOwner = LocalLifecycleOwner.current
                 ContributorsScreen(
                     viewModel = hiltViewModel<ContributorsViewModel>(),
-                    onNavigationIconClick = navController::popBackStack,
+                    onNavigationIconClick = {
+                        handleOnClickIfNotNavigating(
+                            lifecycleOwner,
+                            mainNestedNavController::popBackStack,
```

Comment:
ContributorsScreen is navigated by `mainNestedNavControoler`.
So, changed to use same controller when call `popBackStack()`
By this, app became to pop back from contributors screen.